src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,32 @@
+import { TfiClose } from 'react-icons/tfi';
+
+type PropTypes = {
+  showModal: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+};
+
+// eslint-disable-next-line react/prop-types
```

Comment:
Check it out: works awesome without any FC and there is nothing to disable
![image](https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/66321831/59a21c44-e87f-41ce-9dc3-6ab755bcb735)

Reply:
plus, fix it for 2 components above about city selects